drv/i2c-devices/src/at24csw080.rs

```diff
@@ -64,6 +65,15 @@ impl From<ResponseCode> for Error {
     }
 }
 
+impl From<Error> for ResponseCode {
+    fn from(err: Error) -> Self {
+        match err {
+            Error::I2cError(code) => code,
+            _ => panic!(),
```

Comment:
I wonder if it would make sense to add a catch-all "DeviceSpecific" variant to `ResponseCode`, for safe conversions like this.